### PR TITLE
Miscellaneous cleanups

### PR DESCRIPTION
**Pull request by jouvin**

### CODE REVIEW

File: quattor/client/config.pan [hunk 1]
```diff
@@ -4,7 +4,7 @@
 template quattor/client/config;
 
 variable QUATTOR_CLIENT_CONFIG_SITE ?= null;
-include {if_exists(to_string(QUATTOR_CLIENT_CONFIG_SITE));};
+include if_exists(to_string(QUATTOR_CLIENT_CONFIG_SITE));;
```
Comment (stdweird) [inline quattor/client/config.pan]: double `;` at the end

File: quattor/blockdevices.pan [hunk 2]
```diff
@@ -18,6 +18,9 @@ type vg_string = string with exists ("/system/blockdevices/volume_groups/" + SEL
     || error (SELF + " must be a path relative to "
               + "/system/blockdevices/volume_groups");
 
+type lv_string = string with exists ("/system/blockdevices/logical_volumes/" + SELF)
```
Comment (stdweird) [inline quattor/blockdevices.pan]: is this code from another PR?

**Comment by jouvin** [comment]
Not sure what I made wrong. I rebased on 15.12.0 and fixed the `;;`.

**Comment by jrha** [comment]
Most of the pull request should have been opened against https://github.com/quattor/aii the conflicts arose from the updates to the aii templates from that repository which happen during the release process.

**Pull request merged**